src/EA.Weee.Web/Areas/Admin/ViewModels/Aatf/FacilityViewModelBase.cs

```diff
@@ -44,21 +37,44 @@ public abstract class FacilityViewModelBase : IValidatableObject
 
         public IEnumerable<AatfSize> SizeList { get; set; }
 
-        public FacilityType FacilityType { get; set; }
-
         [Required]
         [Display(Name = "Size")]
         public int SizeValue { get; set; }
 
-        public AatfAddressData SiteAddressData { get; set; }
+        [Display(Name = "Approval date")]
+        [DataType(DataType.Date)]
+        [Required]
+        public DateTime? ApprovalDate { get; set; }
+
+        [Required]
+        [Display(Name = "Compliance year")]
+        public short ComplianceYear { get; set; }
+
+        public FacilityType FacilityType { get; set; }
+
+        public bool ModelValidated { get; private set; }
+
+        public static IEnumerable<string> ValidationMessageDisplayOrder => new List<string>
```

Comment:
Can we add a unit test to check this property and that they are in this order :)

Reply:
Added